clang/test/CodeGen/ms-hotpatch-lto.c

```diff
@@ -0,0 +1,18 @@
+// This verifies that hotpatch function attributes are correctly propagated through LLVM IR when compiling with LTO.
+//
+// RUN: %clang_cl -c --target=x86_64-windows-msvc -O2 /Z7 -fms-hotpatch-functions-file=%S/ms-hotpatch-functions.txt -flto /Fo%t.bc %s
```

Comment:
If you wish, you could also you `split-file` before this line to keep the test self-contained. See https://llvm.org/docs/TestingGuide.html#extra-files

Reply:
Good to know.  I've changed this to a single `echo`, which I think is simpler than using `split-file`.  If it gets any more complicated than that, I'll use `split-file`.